clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -49,6 +49,8 @@ class CIRGenFunction : public CIRGenTypeCache {
 
 public:
   clang::QualType fnRetTy;
```

Comment:
Actually I question the need for `fnRetTy` too, particularly since `fnRetQualTy` and `fnRetTy` are not being set together...